spec/features/authorization_of_conversation_spec.rb

```diff
@@ -27,8 +27,8 @@
     before do
       visit root_path
       click_link "Sign In"
-      fill_in 'Email', with: user.email
-      fill_in 'Password', with: user.password
+      fill_in 'Email', with: conversation.sender.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.